.tool-versions

```diff
@@ -1,2 +1,2 @@
-erlang 20.2.2
 elixir 1.6.1
+erlang 20.2.2
```

Comment:
I flipped the order because there's a bug in asdf-erlang where the install processes ends in a non-0 code, which causes asdf to stop installing the rest of the tools. Moving it to the end will avoid the issue.